packages/react-menu/src/components/MenuItemCheckbox/MenuItemCheckbox.types.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { ComponentProps, ObjectShorthandProps, ShorthandProps } from '@fluentui/react-utils';
+import { MenuItemSelectableProps, MenuItemSelectableState } from '../../selectable/index';
+import { MenuItemProps, MenuItemState } from '../MenuItem/MenuItem.types';
+
+export interface MenuItemCheckboxProps
+  extends ComponentProps,
+    React.HTMLAttributes<HTMLElement>,
+    MenuItemProps,
+    MenuItemSelectableProps {
+  /**
+   * Icon slot rendered before children content
+   */
+  icon?: ShorthandProps;
+
+  /**
+   * Slot for the checkmark indicator
+   */
+  checkmark?: ShorthandProps;
+}
+
+export interface MenuItemCheckboxState extends MenuItemCheckboxProps, MenuItemState, MenuItemSelectableState {
+  ref: React.MutableRefObject<HTMLElement>;
+
+  /**
+   * Icon slot rendered before children content
+   */
+  icon?: ObjectShorthandProps<HTMLElement>;
+
+  /**
+   * Slot for the checkmark indicator
+   */
+  checkmark: ObjectShorthandProps<HTMLElement>;
```

Comment:
```suggestion
  checkmark?: ObjectShorthandProps<HTMLElement>;
```

Is it optional, too?

Reply:
not during state, I haven't actually set the default value here because I don't want to depend on unconverged icons, but the plan will be to have the 'tick' as a default